README.md

```diff
@@ -86,10 +47,10 @@ To accept a certain payment method in your app, you only need to include that pa
 ## Sample Code
 
 ```swift
-// STEP 0: Fetch an ACCESS_TOKEN and ORDER_ID from your server.
+// STEP 0: Enter a CLIENT_ID and fetch ORDER_ID from your server.
```

Comment:
```suggestion
// STEP 0: Fetch a CLIENT_ID and ORDER_ID from your server.
```

Reply:
Do we have a sample cURL or link to a cURL for how merchants can perform a clientID fetch on their server?

Reply:
That's a good point, I don't think we have one for the function Steven wrote on for the merchant server. I can write it in as a comment until we have it in the integration guide. What do you think?

Reply:
A typical merchant won't need to fetch a client ID. It will most likely be injected at compile time using an environment (or hardcoded) variable for most, unless they need to dynamically set their client ID.

Reply:
We'll eventually support multiple merchants in the Demo app e.g. to test connected partner flows, so we need to set it dynamically for PPCP e2e testing.

Reply:
If we expect merchants to hardcode this value, we should document the merchants expected path to enter a client ID vs fetching it on their server, imo

Reply:
[On Android](https://github.com/paypal/Android-SDK/tree/main/docs/CardPayments#2-initiate-the-payments-sdk), we only mention that it's needed to construct a `CoreConfig` object. It is technically up to the merchant, and `clientID` doesn't need to be protected like the `clientSecret` does. Maybe we can do something similar here and eliminate step 0?

Reply:
Do we need this snippet in the README at all? We have this integration covered in our `docs/` section per module.

Reply:
We should be able to remove:
```swift
let paymentConfig = CoreConfig(clientID: "<CLIENT_ID>", environment: .sandbox)
```
is probably all that's needed.

Reply:
I mean this entire section under `## Sample Code` in the README ([link](https://github.com/paypal/iOS-SDK#sample-code)). IMO it can be removed. We don't have an equivalent in Android and the sample code for each flow already lives in `docs/`.

Reply:
👍 agreed.